CHANGES

```diff
@@ -9,6 +9,9 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) The EVP_PKEY_CTX_set_dh_pad() macro has now been converted to a function.
+     [Matt Caswell]
+
```

Comment:
For this kind of change, that's a severely minimalistic entry.
(generally speaking, CHANGES is severely under-edited already)
I would like to see a more fleshed out entry (side note: and appreciate a reminder any time *I* fail to add entries to this file, which I know is a lot of times)

Reply:
> For this kind of change, that's a severely minimalistic entry.

I'm struggling to think what extra detail you want me to add

Reply:
Uhmmm, this PR does a bit more than converting `EVP_PKEY_CTX_set_dh_pad()`, doesn't it?  I struggle to see how one can *not* see that this needs fleshing out...

Reply:
Ah! I see what you are getting at. I would see that as an entirely different CHANGES entry to this one, i.e. not so much fleshing this one out - but adding a new one.

Reply:
I'm totally fine with that.

Reply:
I've added a new CHANGES entry